build/Publish.targets

```diff
@@ -175,15 +175,23 @@
                     AccountKey="$(PublishBlobFeedKey)"
                     ItemsToPush="@(PackagesToPublishToTransport)"
```

Comment:
We should also update the ManifestArtifactData metadata on these items to `NonShipping=true` for our noship packages

Reply:
Ahh, oops